docs/hbase.md

```diff
@@ -0,0 +1,89 @@
+# `hbase` Metrics Receiver
+
+The `hbase` metrics receiver can fetch stats from a Hbase server's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html).
```

Comment:
Please explain in the docs how the cluster metrics are collected (does it only collect the local region server's metrics? or metrics for all the region servers in the cluster?).

Reply:
Added